src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -588,8 +588,7 @@ void PolicyManagerImpl::CheckPermissions(const PTString& app_id,
     policy_table::FunctionalGroupings functional_groupings;
     cache_->GetFunctionalGroupings(functional_groupings);
 
-    policy_table::Strings app_groups =
```

Comment:
Changes to certificate processing via policies should be targeted via the bug fix branches #2190 and #2191